pkg/fakerp/config.go

```diff
@@ -65,3 +71,77 @@ func NewConfig(log *logrus.Entry) (*Config, error) {
 	os.Setenv("AZURE_REGION", c.Region)
 	return &c, nil
 }
+
+func (s *Server) configure() (*api.PluginConfig, error) {
+	tc := api.TestConfig{
+		RunningUnderTest:      os.Getenv("RUNNING_UNDER_TEST") != "",
+		ImageResourceGroup:    os.Getenv("IMAGE_RESOURCEGROUP"),
+		ImageResourceName:     os.Getenv("IMAGE_RESOURCENAME"),
+		DeployOS:              os.Getenv("DEPLOY_OS"),
+		ImageOffer:            os.Getenv("IMAGE_OFFER"),
+		ImageVersion:          os.Getenv("IMAGE_VERSION"),
+		ORegURL:               os.Getenv("OREG_URL"),
+		EtcdBackupImage:       os.Getenv("ETCDBACKUP_IMAGE"),
+		AzureControllersImage: os.Getenv("AZURE_CONTROLLERS_IMAGE"),
+	}
+
+	// populate geneva artifacts
```

Comment:
I dont like this part below :/

Reply:
removing all the fileExist stuff would probably make it less unpleasant.

Reply:
maybe a separate function which returns a *GenevaConfig as well?

Reply:
will do this with metrics PR.